recipes/sknw/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "sknw" %}
+{% set version = "0.12" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "fafdbbf93b9f579c36de4e9d7602cd7b23530841a60575ed009c1bbf3f820f25" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
```

Comment:
This looks like a candidate for `noarch: python`:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages